src/pages/AddMember.js

```diff
@@ -144,32 +186,45 @@ const MemberForm = () => {
 
        <div className='form-row'>
 
-       <div className='form-group col-5'>
-           <label htmlFor="profile_img" style={{marginTop: '2%'}}>Profile Image</label>
-           <input type="text" class="form-control" id="profile_img" value={member.profile_img} style={{marginLeft: '2%', marginTop: '.5%'}}
+       <div className='form-group col-11'>
+           <label htmlFor="profile_img" style={{marginTop: '2%', marginLeft: '7%', fontWeight: '800'}}>
+             Profile Image
+           </label>
+           <input type="text" class="form-control" id="profile_img" value={member.profile_img} 
+           style={{marginLeft: '5%', marginTop: '.5%'}}
            onChange={event => {
              setMember({...member, profile_img: event.target.value});
            }}  />
          </div>
 
-         <div className='form-group col-6'>
-           <label htmlFor="languages" style={{marginTop: '2%'}}>Languages</label>
-           <input type="text" class="form-control" id="languages" value={member.languages} style={{marginLeft: '2%'}}
+       </div>
+
+       <div className='form-row'>
+
+         <div className='form-group col-11'>
+           <label htmlFor="languages" style={{marginTop: '2%', marginLeft: '7%', fontWeight: '800'}}>
+             Languages
+           </label>
+           <input type="text" class="form-control" id="languages" value={member.languages} 
+           style={{marginLeft: '5%'}}
            onChange={event => {
              setMember({...member, languages: [event.target.value]});
            }}  />
          </div>
         
        </div>
 
-       <button type="submit" class="btn btn-dark btn-outline-light btn-block" style={{marginBottom: '1%', marginLeft: '1%', marginRight: '1%'}}>Save Member</button>
+       <button type="submit" class="btn btn-dark btn-outline-light" 
+       style={{marginTop: '2%', marginLeft: '35%', marginRight: '1%', width: '30%'}}>
+         Save Member
+       </button>
 
       </div>
        </form>
 
 
       
-       <div className="row">
+            <div className="row">
                 <div className="col d-flex flex-row flex-wrap">
                     {members.map((m, i) =>{
                         return(
```

Comment:
On this file we want the pictures to be consistent with the rest of the format pictures in members so we need to change this file.